### PR TITLE
enhance integration test

### PR DESCRIPTION
**Pull request by nittka**
Here are some first possible improvements for the integration tests.

* I added an extra (working) directory where failing integration test files are copied. That way they can be inspected more easily (even if the JUnit-View does not contain the test results anymore).

* The JUnit view shows only the short file location, not the full path (easier to read)

* I added a second assertion in Integration.java. From the file string I extract a list of expected assignments (same heuristic as in the markup hack - "id = ..." at the beginning of a line) and check this list against the assignments actually found in the model. In the first phase of improving the grammar, this might help to find errors, as definitions may not be referenced and hence do not cause a linking error.

* I added a folder "regression" for test case files of a particular version. If we go to 2.19.X, we still want old fixes to work, but init- and snippet-files might be updated. TrillTest is a possible example of such a regression test.

For Xtext tests I quite like using Xtend. Simple models to test can be defined within the xtend file as rich string. This is more convenient than having all input as external file or having to create the multi-line string in Java. Also the test infrastructure might be updated with cool new features of the xtext.junit-framework (run with GuiceRunner, injecting parse and validation helpers etc., see e.g. https://github.com/AKSW/Xturtle/blob/develop/de.itemis.tooling.xturtle.tests/src/de/itemis/tooling/xturtle/parsing/ParserTest.xtend)

### CODE REVIEW

**Comment by thSoft** [comment]
I think these are all great improvements. I encourage you to use Xtend, I prefer it also.

File: org.elysium.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -8,4 +8,19 @@ Require-Bundle: org.eclipse.xtext;bundle-version="1.0.1",
...
+ javax.util;bundle-version="0.0.0",
+ org.elysium.ui,
+ org.eclipse.core.runtime,
+ org.eclipse.xtext.junit4,
+ org.eclipse.ui.workbench;resolution:=optional,
+ com.google.guava,
+ org.eclipse.xtext.xbase.lib,
+ org.eclipse.xtend.lib,
+ org.eclipse.xtend.lib.macro
+Export-Package: org.elysium
```
Comment (thSoft) [inline org.elysium.test/META-INF/MANIFEST.MF]: There is a Plug-in Problem here on my machine: Package 'org.elysium' does not exist in this plug-in

File: org.elysium.test/src/org/elysium/test/regression/TrillTest.xtend [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package org.elysium.test.regression
+
+import org.junit.runner.RunWith
+import org.eclipse.xtext.junit4.XtextRunner
+import org.eclipse.xtext.junit4.InjectWith
+import org.elysium.LilyPondInjectorProvider
```
Comment (thSoft) [inline org.elysium.test/src/org/elysium/test/regression/TrillTest.xtend]: I have an Xtend error here: org.elysium.LilyPondInjectorProvider cannot be resolved to a type.

**Comment by thSoft** [comment]
Thanks for the contribution!

**Comment by nittka** [comment]
I added the junit fragment to the workflow file. You have to rerun the language infrastructure generation. Then all compile errors should go away.

**Pull request merged**